doc/guides/modules.rst

```diff
@@ -158,6 +158,52 @@ labels corresponding to each module, when applicable.
      Bug reports in modules, mirroring upstream project bug reports, are
      allowed but shall not block Zephyr releases.
 
+Licensing requirements and policies
+***********************************
+
+.. _modules_licensing:
+
+All source files in a module's code base require to include a license header,
+unless there exist a **main license file** that covers the un-licensed files.
+
+Main license files shall be added in the module code base by Zephyr
+developers; instead, only if they exist, as part of the external project
+and they contain a permissive OSI-compliant license. Main license files
```

Comment:
I am not a native English speaker so perhaps this sounds correct to native one but I have difficulty understanding what this sentence is actually saying. Could this one simplified a bit?

Reply:
Yeap, fixed that as well (removed "instead"). thank you